src/core/qgslegendsettings.h

```diff
@@ -178,6 +178,20 @@ class CORE_EXPORT QgsLegendSettings
      */
     void setMapScale( double scale ) { mMapScale = scale; }
 
+    /**
+     * Returns the factor of map units per pixel for symbols with size given in map units calculated by dpi and mmPerMapUnit
+     * \see setMapUnitsPerPixel()
+     * \since QGIS 3.6
```

Comment:
Last minute review: `QGIS 3.8`

Reply:
Of course. I was optimistic on Friday :-)

Reply:
But I'll do a backport to 3.6 and 3.4, right?

Reply:
> But I'll do a backport to 3.6 and 3.4, right?

It would be great yes :).